src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -871,6 +871,9 @@ private static function get_wp_templates_author_text_field( $template_object ) {
 				}
 				return $author->get( 'display_name' );
 		}
+
+		// Ensure that the method always returns a string.
```

Comment:
```suggestion
		/*
		 * Ensure a string data type is always returned as the default and
		 * a fail-safe in case `$original_source` is not one of the cases.
		 */
```

Reply:
I'm surprised I overlooked that extra slash in such an obvious spot 🙃. Thanks for fixing it.